test/src/main/java/org/apache/accumulo/test/functional/SplitRecoveryIT.java

```diff
@@ -189,12 +187,26 @@ private void runSplitRecoveryTest(ServerContext context, int failPoint, String m
 
   private static Map<Long,List<ReferencedTabletFile>> getBulkFilesLoaded(ServerContext context,
       KeyExtent extent) {
-    Map<Long,List<ReferencedTabletFile>> bulkFiles = new HashMap<>();
 
-    context.getAmple().readTablet(extent).getLoaded().forEach((path, txid) -> bulkFiles
-        .computeIfAbsent(txid, k -> new ArrayList<>()).add(path.getTabletFile()));
+    // Ample is not used here because it does not recognize some of the old columns that this
+    // upgrade code is dealing with.
+    try (Scanner scanner =
+        context.createScanner(AccumuloTable.METADATA.tableName(), Authorizations.EMPTY)) {
+      scanner.setRange(extent.toMetaRange());
+
+      Map<Long,List<ReferencedTabletFile>> bulkFiles = new HashMap<>();
+      for (var entry : scanner) {
+        if (entry.getKey().getColumnFamily().equals(BulkFileColumnFamily.NAME)) {
+          var path = new StoredTabletFile(entry.getKey().getColumnQualifier().toString());
+          var txid = BulkFileColumnFamily.getBulkLoadTid(entry.getValue());
+          bulkFiles.computeIfAbsent(txid, k -> new ArrayList<>()).add(path.getTabletFile());
+        }
+      }
```

Comment:
So, this looks like the real change here, the other changes are due to the method moves. What is the old column here?

Reply:
[These columns](https://github.com/apache/accumulo/blob/5d12597b4dedb60524775d0411405bafc3ee86ce/core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataSchema.java#L427-L433) are handled on upgrade by the code this class is testing.  If Ample sees those columns it throws an exception.